src/System.Text.Encoding/tests/NegativeEncodingTests.cs

```diff
@@ -36,11 +36,19 @@ public static void GetByteCount_Invalid(Encoding encoding)
             Assert.Throws<ArgumentOutOfRangeException>("chars", () => encoding.GetByteCount(new char[3], 2, 2));
             Assert.Throws<ArgumentOutOfRangeException>("chars", () => encoding.GetByteCount(new char[3], 3, 1));
             Assert.Throws<ArgumentOutOfRangeException>("chars", () => encoding.GetByteCount(new char[3], 4, 0));
+
+            char[] chars = new char[3];
+            fixed (char* pChars = chars)
+            {
+                char* pCharsLocal = pChars;
```

Comment:
I get a compiler error if I use `pChars` as I can't use something in a fixed statement in a lambda expression (i.e. the `Assert.Throws<...>(() => ...)`)